### DISCUSSION TITLE
I want see all achievements

### DISCUSSION THREAD

**Discussion by lgrachov** [community/community#20442] [Other Feature Feedback, Questions, & Ideas] [answered]
Please show me all available achievements

**Comment by thuongtruong1009** [comment]
Hi @lgrachov 
This bellow are some my available achievements
![image](https://user-images.githubusercontent.com/71834167/178768378-13386759-3969-4589-a66f-6e469d50b59d.png)

**Reply by thuongtruong1009** [reply]
Hi @lgrachov 
Can you ***mark them as answers*** to help more people can see it, please! Thank you very much!
Have a nice day!

**Reply by amendezm** [reply]
<img width="61" alt="image" src="https://user-images.githubusercontent.com/26444448/178772355-f1d4c5c5-f09d-40f4-9757-70c3fcfea121.png">

@thuongtruong1009 How did you obtain this?

**Reply by hqyuh** [reply]
I want to know too

**Reply by thuongtruong1009** [reply]
Hi @amendezm and @hqyuh 
I have just posted at here, https://github.com/github-community/community/discussions/20443
Follow me if useful for you! Thank you very much!

**Reply by FahdSeddik** [reply]
> 

you can get it by getting at least 16 stars on a repo you made

**Reply by thuongtruong1009** [reply]
> > 
> 
> you can get it by getting at least 16 stars on a repo you made

yes. least 16 stars is level1, and I predict that will have more level for get more stars

**Reply by amendezm** [reply]
Thank you @thuongtruong1009

**Reply by MohitGupta121** [reply]
@thuongtruong1009 buddy it's a shock to see your profile. You are really from 1990 at Github? That's awesome.

**Comment by FahdSeddik** [comment]
Hey!
You can check all achievements and how to get them in this [link](https://github.com/Schweinepriester/github-profile-achievements). Hope it helps!

**Comment by Iglesys347** [comment]
<img width="148" alt="pair-extraordinaire-default" src="https://user-images.githubusercontent.com/22981389/178778068-243087ee-d054-4dd5-aa1f-daa00d1269ec.png">
There is also this one you can obtain by coauthoring commits.

**Reply by rahmatsubandi** [reply]
Can you provide detailed steps to get this?

**Reply by lgrachov** [reply]
No

**Reply by thenithinbalaji** [reply]
@rahmatsubandi check this out
https://github.com/Schweinepriester/github-profile-achievements/discussions/127

**Comment by mrcsxsiq** [comment, marked as answer]
Try checking out this one!
https://github.com/Schweinepriester/github-profile-achievements

**Comment by tofuwu-dev** [comment]
There are plenty of answers for it...
https://github.com/Schweinepriester/github-profile-achievements

**Comment by rodukov** [comment]
View it here: https://github.com/Schweinepriester/github-profile-achievements

**Comment by diegoleondev** [comment]
👋 Hi @lgrachov, These are all the achievements so far:

## Achievements

| Title | Badge | Earnable? | Earned by |
| --- | --- | --- | --- |
Pair Extraordinaire | ![Pair Extraordinaire Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/pair-extraordinaire-default.png) | ✔️ | Coauthored in a merged pull request
Quickdraw | ![Quickdraw Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/quickdraw-default.png) | ✔️ | Closed an issue or a pull request within 5 min of opening
Starstruck | ![Starstruck Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/starstruck-default.png) | ✔️ | Created a repository that has 16 stars
||| 
Galaxy Brain | ![Galaxy Brain Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/galaxy-brain-default.png) | ✔️ | 2 accepted answers
Pull Shark | ![Pull Shark Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/pull-shark-default.png) | ✔️ | 2 pull requests merged
YOLO | ![YOLO Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/yolo-default.png) | ✔️ | Merged a pull request without code review
||| 
Arctic Code Vault Contributor | ![Arctic Code Vault Contributor Achievement Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/arctic-code-vault-contributor-default.png) | ❌ | Contributed code to repositories in the [2020 GitHub Archive Program](https://archiveprogram.github.com/)
Public Sponsor | ![GitHub Sponsor Achievement Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/public-sponsor-default.png) | ✔️ | Sponsoring open source work via [GitHub Sponsors](https://github.com/sponsors)
Mars 2020 Contributor | ![Mars 2020 Helicopter Contributor Achievement Badge](https://github.com/Schweinepriester/github-profile-achievements/raw/main/images/mars-2020-contributor-default.png) | ❌ | Contributed code to repositories used in the [Mars 2020 Helicopter Mission](https://github.com/readme/nasa-ingenuity-helicopter)

You can supplement the information in the following github article: https://github.com/Schweinepriester/github-profile-achievements/blob/main/README.md

>**Note**
>Please **Mark as answer** if it was helpful.

**Comment by MuhammadRestu999** [comment]
Hi @lgrachov 👋🏼 

You can see all achievements from this link [Schweinepriester/github-profile-achievements](https://github.com/Schweinepriester/github-profile-achievements#achievements)

**Comment by fahimy143** [comment]
hi @lgrachov 
check out this
https://github.com/Schweinepriester/github-profile-achievements
